CHANGELOG.md

```diff
@@ -24,6 +24,26 @@ All notable changes to the code converter will be documented here.
 * Correctly convert ambiguous names ([#332](https://github.com/icsharpcode/CodeConverter/pull/332))
 * Ensure correct visibility for constructors ([#422](https://github.com/icsharpcode/CodeConverter/pull/422))
 * Ensure casing is correct for namespaces ([#421](https://github.com/icsharpcode/CodeConverter/pull/421))
+* Convert implicit object->string cast correctly (#365)
```

Comment:
Think you found the downside to `merge=union` here, @GrahamTheCoder 😉

Reply:
Ah yes. I'll fix it on master